guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,570 @@
+    val config = ConfigFactory.load("custom.conf")
+    val actorSystem = ActorSystem(actorSystemName, config)
+  }
+```
+
+### Publishing to Docker registry
+
+Next we will containerize the application and publish its Docker image. Please proceed with either [SBT instructions](#Publishing-to-Docker-registry---SBT) or [Docker instructions](#Publishing-to-Docker-registry---Maven) accordingly.
+
+
+### Publishing to Docker registry - SBT
+
+We will be using the [SBT Native Packager](http://www.scala-sbt.org/sbt-native-packager/) plugin to containerize the application.
+
+Enable SBT Native Packager in your project by adding the following line in the `project/plugins.sbt`:
+
+```
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0")
+```
+
+#### Multi-module project
+
+Follow this section if your application is part of a multi-module SBT project.
+
+Enable the [JavaAppPackaging](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/index.html) plugin provided by SBT Native Packager by adding the `enablePlugins` instruction to the module declaration in the `build.sbt`:
+
+```
+lazy val myApp = project("my-app")
+  // Add the following line to the module definition
+  .enablePlugins(JavaAppPackaging)
+```
+
+Alternatively you may choose to enable [JavaServerAppPackaging](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html) instead:
+
+```
+lazy val myApp = project("my-app")
+  // Add the following line to the module definition
+  .enablePlugins(JavaServerAppPackaging)
+```
+
+The `JavaServerAppPackaging` plugin provides all the features provided by the `JavaAppPackaging` plugin with some additional [server features](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html#features) such as daemon user/group support and support for `/etc/default`.
+
+Please note that you must choose to enable either `JavaAppPackaging` or `JavaServerAppPackaging` - you _can't_ enable both.
+
+We will now configure the Docker settings required to containerize the application.
+
+Append the Akka clustering related sytem properties to the `dockerEntrypoint` setting within the module settings:
+
+```
+lazy val myApp = project("my-app")
+  .settings(
+    // Add the following line within the module settings
+    dockerEntrypoint ++= """-Dplay.crypto.secret="${APPLICATION_SECRET:-none}" -Dplay.akka.actor-system="${AKKA_ACTOR_SYSTEM_NAME:-friendservice-v1}" -Dhttp.address="$FRIENDSERVICE_BIND_IP" -Dhttp.port="$FRIENDSERVICE_BIND_PORT" -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$AKKA_REMOTING_BIND_HOST" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" -Dakka.io.dns.resolver=async-dns -Dakka.io.dns.async-dns.resolve-srv=true -Dakka.io.dns.async-dns.resolv-conf=on""".split(" ").toSeq
```

Comment:
Shouldn't have the `play.` entries here right? As this is for a regular akka application